source/MRI.md

```diff
@@ -3,27 +3,28 @@
+ The parameters of the functional MRI sequence relevant for data analysis can be found in the [BIDS dataset](https://git.unf-montreal.ca/neuromod/), and included metadata. The functional acquisition parameters are all identical to the one used in the HCP-trt dataset. The Siemens exam card can be found [here](./_static/mri/functional_protocol_HCP-trt.pdf), and is briefly recapitulated below. Functional MRI data was acquired using an accelerated simultaneous multi-slice, gradient echo-planar imaging sequence [(Xu et al., 2013)](http://www.ncbi.nlm.nih.gov/pubmed/23899722) developed at the Center for Magnetic Resonance Research (CMRR) University of Minnesota, as part of the Human Connectome Project [(Glasser et al., 2016)](https://www.nature.com/articles/nn.4361). The sequence is available on the Siemens PRISMA scanner at UNF through a concept to production (C2P) agreement, and was used with the following parameters: slice acceleration factor = 4, TR = 1.49 s, TE = 37 ms, flip angle = 52 degrees, voxel size = 2 mm x 2 mm x 2 mm, 60 slices, acquisition matrix 96x96. In each session, a short acquisition (3 volumes) with reversed phase encoding direction was run to allow retrospective correction of B0 field inhomogeneity-induced distortion.
 
 ### Brain anatomical sequences
 
  The parameters of the brain anatomical MRI sequences relevant for data analysis can be found in the [BIDS dataset](https://git.unf-montreal.ca/neuromod/), and included metadata. The acquisition parameters are identical for all anatomical sessions. The Siemens pdf exam card of the anatomical sessions can be found [here](./_static/mri/anatomical_protocol_2019-01-22.pdf), and is briefly recapitulated below. A standard (brain) anatomical session starts with a 21 s localizer scan, and then includes the following sequences:
-  * T1-weighted MPRAGE sequence (duration 6:38 mn, TR=, TE=)
-  * T2-weighted sequence (duration 5:57 mn, ...)
-  * Diffusion-weighted sequence (duration 4:04 mn, ...)
-  * gradient-echo magnetization-transfer sequence (duration 3:34 mn, ...)
-  * gradient-echo proton density sequence (duration 3:34 mn, ...)
+  * T1-weighted MPRAGE 3D sagittal sequence (duration 6:38 min, TR = 2.4 s, TE = 2.2 ms, voxel size = 0.8 mm isotropic, R=2 acceleration)
```

Comment:
is it the                               gre_3d_T1w scan that can be found in the https://github.com/courtois-neuromod/cneuromod_docs/blob/master/source/_static/mri/anatomical_protocol_2019-01-22.pdf ?

Reply:
> @jcohenadad is it the gre_3d_T1w or the t1_mp2rage scan that can be found in the https://github.com/courtois-neuromod/cneuromod_docs/blob/master/source/_static/mri/anatomical_protocol_2019-01-22.pdf ?

it's the acq-0.8mm_T1w

but i've already entered and verified this, so it should be fine